parametric-eq/src/eq_core/svf.rs

```diff
@@ -35,8 +35,8 @@ pub struct SVFCoefficients<T> {
 }
 
 impl SVFCoefficients<f64> {
-    pub fn get_amplitude(self, f_hz: f64) -> f64 {
-        //TODO gain up MESA
+    pub fn get_bode_sample(self, f_hz: f64) -> Complex<f64> {
+        //Use y.norm() for amplitude and -y.arg().to_degrees() for phase. Add to combine phase.
```

Comment:
Negating the output phase should not be necessary. This is caused by the sign of the exponents of `z` being incorrect.

I would suggest the following:

- Keep this line as is:
  ```rust
  let z = (-TAU * f_hz * imag / self.fs).exp();
  ```
  The negation means you actually get `z^{-1}`, but that's fine. To optimize, you could also do `cos(w) + isin(w)` explicitly here, since scalar exponentiation is not necessary.
- Use `z` directly instead of its reciprocal:
  ```rust
  let gpow2 = self.g * self.g; // this too, why not
  let z_n1 = z;
  let z_n2 = z * z;
  ```
  This avoids two costly complex `powf`s, while also removing the need to negate the phase of the result.